sfs/time/drivingfunction.py

```diff
@@ -6,6 +6,7 @@
 from __future__ import division
 import numpy as np
 from numpy.core.umath_tests import inner1d  # element-wise inner product
+import scipy.signal as sig
```

Comment:
I'm not sure if `sig` is a good name.
Typically, one would use `from scipy import signal`, but of course in our case we are already using `signal` as variable name ...

What about importing the individual functions by name?

    from scipy.signal import besselap, sosfilt, zpk2sos

Reply:
I agree. Now only those three functions are imported. Thanks!